### PR TITLE
create list and image component

### PR DESCRIPTION
**Pull request by KevinBlbd**

### CODE REVIEW

File: components/Gallery.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import styled from "styled-components";
+import { useState, useEffect } from "react";
+import Card from "./Card";
+
+const StyleNoneList = styled.li`
+  list-style: none;
+`;
+
+export default function Gallery() {
+  const pieces = [
```
Comment (botacode-gh) [inline components/Gallery.js]: [Minor note I learned](https://github.com/botacode-gh/capstone-project/pull/10#discussion_r1176422259) from @onemanwenttomow: take the const out of the function so it doesn't need to be redeclared with every re-render

**Review by botacode-gh — COMMENTED** [review]
Left a comment, otherwise LGTM!

File: components/Card.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  margin-top: 1rem;
+  margin-left: -1rem;
+  @media screen and (max-width: 768px) {
+    margin-bottom: 0rem;
+  }
+`;
+
+export default function Card({ id, location, type, image }) {
+  return (
+    <>
```
Comment (BenRodProd) [inline components/Card.js]: If you are only returning this one element, you do not need <> </> there, this is what i am told all the time...

Comment (BenRodProd) [reply components/Card.js]: rest looks good...

File: components/Card.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import GlobalStyle from "../styles";
+import Image from "next/image";
+import styled from "styled-components";
```
Comment (botacode-gh) [inline components/Card.js:1]: It's conventional to import external stuff first:
```jsx
import styled from "styled-components";
import Image from "next/image";
import GlobalStyle from "../styles";
```

**Pull request merged**